src/main/java/be/syntra/devshop/DevshopFront/services/CartServiceImpl.java

```diff
@@ -39,4 +43,55 @@ public void addToCart(Product product) {
         getCart().setProducts(productList);
         log.info("addToCart() -> {}", currentCart.getProducts().size());
     }
+
+    @Override
+    public StatusNotification payCart(CartDto cartDto) {
+        // cartDto is send to backend through rest template url with the users name which is the email.
```

Comment:
incomplete method. This is from DEV-013? Please remove it for merge-safety's sake

Reply:
removed, will be in DEV-013